indexer/map_object.cpp

```diff
@@ -188,6 +190,33 @@ string MapObject::GetBuildingLevels() const
   return m_metadata.Get(feature::Metadata::FMD_BUILDING_LEVELS);
 }
 
+vector<string> MapObject::GetBusRoutes() const
+{
```

Comment:
Or, even better, create a single method in MetadataObject, like:
```c++
vector<string> GetRoutes(RouteType type);
```
where RouteType can be one of:
```
ROUTE_TYPE_BUS,
ROUTE_TYPE_TRAM,
ROUTE_TYPE_TROLLEYBUS
```